### PR TITLE
[new package] trunk 0.20.3

### PR DESCRIPTION
**Pull request by podsvirov**

### CODE REVIEW

File: mingw-w64-trunk/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+pkgrel=1
+pkgdesc='Build, bundle & ship your Rust WASM application to the web (mingw-w64)'
+arch=('any')
+mingw_arch=('mingw64' 'ucrt64' 'clang64' 'clangarm64')
+url='https://trunkrs.dev/'
+msys2_repository_url='https://github.com/trunk-rs/trunk'
+license=('spdx:MIT OR Apache-2.0')
+depends=("${MINGW_PACKAGE_PREFIX}-openssl"
+         "${MINGW_PACKAGE_PREFIX}-rust-wasm")
+makedepends=("${MINGW_PACKAGE_PREFIX}-pkg-config"
```
Comment (ognevny) [inline mingw-w64-trunk/PKGBUILD:16]:
```suggestion
makedepends=("${MINGW_PACKAGE_PREFIX}-pkgconf"
```

File: mingw-w64-trunk/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+sha256sums=('388d42cef7b8e78081c3371f31e11db6aba69b65c5c89ccf2f0d537922e4ddce')
+
+prepare() {
+  cd "${srcdir}/${_realname}-${pkgver}"
+
+  cargo fetch --locked --target "$(rustc -vV | sed -n 's/host: //p')"
+}
+
+build() {
+  cd "${srcdir}"
```
Comment (ognevny) [inline mingw-w64-trunk/PKGBUILD:28]: that's not required, every functions start at $srcdir

**Comment by ognevny** [comment]
why not using `rustls`? nothing bad with using openssl, but pure rust implementation looks better as there won't be problems in case of C libraries' major updates

**Comment by podsvirov** [comment]
> why not using `rustls`? nothing bad with using openssl, but pure rust implementation looks better as there won't be problems in case of C libraries' major updates

My work based on `AUR` [recipe](https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=trunk&id=2456a09f3039fa50613c141c9713711fcc3a1a0d#n18). I suggest using `openssl` for recipe consistency.

**Comment by ognevny** [comment]
better use squash and merge option next time, so unwanted `update <file>` commits won't spam the history

**Comment by podsvirov** [comment]
> better use squash and merge option next time, so unwanted `update <file>` commits won't spam the history

Ok.

**Pull request merged**